MonoGame.Framework/Point.cs

```diff
@@ -77,6 +77,37 @@ public Point(int x, int y)
             this.Y = y;
         }
 
+        /// <summary>
+        /// Creates a new instance of <see cref="Point"/> struct, with the specified position.
+        /// </summary>
+        /// <param name="value">The x and y coordinates in 2d-space.</param>
+        public Point(int value)
+        {
+            this.X = value;
+            this.Y = value;
+        }
+
+        /// <summary>
+        /// Creates a new instance of <see cref="Point"/> struct, with the specified position.
+        /// </summary>
+        /// <param name="x">The x coordinate in 2d-space.</param>
+        /// <param name="y">The y coordinate in 2d-space.</param>
+        public Point(float x, float y)
```

Comment:
I don't mind mimicking the single value ctor in `Point` to match `Vector2`.  It is a shortcut that could eliminate a bug and makes it clear that the coder indented `X` and `Y` to match.